cypress/e2e/smoke/aboutUs.cy.js

```diff
@@ -0,0 +1,23 @@
+import '@testing-library/cypress/add-commands'
+
+describe('About us page', () => {
+
+  beforeEach(() => {
```

Comment:
We can remove the `beforeEach` if no longer needed
```suggestion
  beforeEach(() => {
```